resources/views/settings/labels.blade.php

```diff
@@ -313,6 +313,26 @@ class="col-md-4"
                                     <p class="help-block">{{ trans('admin/settings/general.label2_2d_target_help') }}</p>
                                 </div>
                             </div>
+                            <!-- Field offset -->
+                            <div class="form-group{{ $errors->has('label2_empty_row_count') ? ' has-error' : '' }}">
+                                <div class="col-md-3 text-right">
+                                    <label for="label2_empty_row_count" class="control-label">{{ trans('admin/settings/general.empty_row_count') }}</label>
+                                </div>
+                                <div class="col-md-9">
+                                        <input
+                                                class="form-control col-md-2"
+                                                aria-label="Empty Row Count"
+                                                name="label2_empty_row_count"
+                                                type="number"
+                                                id="label2_empty_row_count"
+                                                min="0"
+                                                max="5"
```

Comment:
It would be better if this is dynamic but I think it's ok to keep it static for now.